dotcom-rendering/src/components/Nav/Nav.tsx

```diff
@@ -1,226 +0,0 @@
-import { css, Global } from '@emotion/react';
-import type { Pillar } from '@guardian/libs';
-import { until, visuallyHidden } from '@guardian/source/foundations';
-import type { EditionId } from '../../lib/edition';
-import { clearFix } from '../../lib/mixins';
-import type { NavType } from '../../model/extract-nav';
-import { GuardianRoundel } from '../GuardianRoundel';
-import { InteractiveSupportButton } from '../InteractiveSupportButton.importable';
```

Comment:
Can the `GuardianRoundel` and the `InteractiveSupportButton` components be removed also, unless you have plans for them in the new header? It doesn't look like they're used elsewhere.